client/src/components/menubar/index.tsx

```diff
@@ -33,13 +32,6 @@ interface StateProps {
   showCentroidLabels: RootState["centroidLabels"]["showLabels"];
   categoricalSelection: RootState["categoricalSelection"];
   screenCap: RootState["controls"]["screenCap"];
-  imageUnderlay: RootState["controls"]["imageUnderlay"];
-  // eslint-disable-next-line react/no-unused-prop-types -- used in shouldShowOpenseadragon
-  layoutChoice: RootState["layoutChoice"];
-  // eslint-disable-next-line react/no-unused-prop-types -- used in shouldShowOpenseadragon
-  config: RootState["config"];
-  // eslint-disable-next-line react/no-unused-prop-types -- used in shouldShowOpenseadragon
-  panelEmbedding: RootState["panelEmbedding"];
```

Comment:
moved the image toggle button to `Embedding` component, so we no longer need these props here